pulsarbot/entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
It looks like all of these files are without ASF license header

Reply:
it is a new repo and we haven't setup license header checks yet.